.github/workflows/test.yaml

```diff
@@ -4,7 +4,7 @@ on:
   push:
     branches: ["master"]
   pull_request:
-    branches: ["master"]
+    branches: ["master", "0.22.x"]
```

Comment:
Nit: just noting that you can use a filter here if we want to stick to this branch naming scheme... I think something like `"0.*.x"` or `0.**` if I understand this correctly https://docs.github.com/en/actions/writing-workflows/workflow-syntax-for-github-actions#using-filters

Reply:
Clicking through, I found [this cheat sheet](https://docs.github.com/en/actions/writing-workflows/workflow-syntax-for-github-actions#filter-pattern-cheat-sheet) and tried to write a more precise filter.